src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs

```diff
@@ -13,6 +13,7 @@ namespace System.Net.Sockets
 {
     public partial class SocketAsyncEventArgs : EventArgs, IDisposable
     {
+        private const int MaxStackWSABuffers = 16;
```

Comment:
Nit:
```suggestion
        private const int MaxStackWSABuffers = 16;

```

Reply:
Wasn't sure what number to use here; 16 seemed ok 😅